libnavigation-base/src/main/res/values-uz/strings.xml

```diff
@@ -1,15 +1,16 @@
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
 
     <!-- Time formatting -->
-    <plurals name="mapbox_number_of_days">
+    <plurals name="mapbox_number_of_days" tools:ignore="MissingTranslation">
         <item quantity="one">kun</item>
```

Comment:
@1ec5 do you know why `one` is not showing up in Transifex? I tried to add it manually but couldn't either 🤔

Reply:
Not sure. Transifex uses CLDR data for pluralization; CLDR says [Uzbek has `one` and `other` cardinal forms](https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html#uz). Is this affecting just Uzbek or other languages as well?

Reply:
Yeah for some reason I'm only running into issues with Uzbek 🤔

Reply:
Back when Transifex was open-source, it had [incorrect plural rules for Uzbek](https://github.com/transifex/transifex-old-core/blob/b67c356f91ddf5e1a5b74dca30124ae051c2c4e2/transifex/languages/fixtures/all_languages.json#L2216-L2219). They periodically update plural rule data based on new CLDR releases, with the next update [coming on May 20](https://community.transifex.com/t/important-update-languages-plural-rules/2832). But it looks like this old data never got looked at.

Reply:
Ok, so it seems nothing much we can do from our side right now. Thanks for following up @1ec5 🙇‍♂️